### PR TITLE
Fix Issue

### LINKED ISSUE #58: Rename TLS certs to match other roles
Other roles create certificates as well but give them a more descriptive name like `[hostname].p12`.

It doesn't really matter, which way we go but we should find a universal one *before* we publish the role.

### PR DESCRIPTION
**Pull request by Nathaniel-Donahue**

### CODE REVIEW

File: tasks/logstash-security.yml [hunk 1]
```diff
@@ -44,7 +44,7 @@
 - name: Copy the certificate to actual node
   copy:
     src: "/tmp/{{ ansible_hostname }}.p12"
-    dest: "/etc/logstash/certs/cert.p12"
+    dest: "/etc/logstash/certs/*"
```
Comment (widhalmt) [inline tasks/logstash-security.yml:47]:
```suggestion
    dest: "/etc/logstash/certs"
```

File: templates/elasticsearch-output.conf.j2 [hunk 2]
```diff
@@ -2,7 +2,7 @@ output {
   elasticsearch {
     hosts => [ {% for host in logstash_elasticsearch %}"{{ host }}:9200"{% if not loop.last %},{% endif %}{% endfor %}]
 {% if elastic_stack_full_stack | bool and logstash_security | bool and elastic_variant == "elastic" %}
-    keystore => "/etc/logstash/certs/cert.p12"
+    keystore => "/etc/logstash/certs/"
```
Comment (widhalmt) [inline templates/elasticsearch-output.conf.j2:5]:
```suggestion
    keystore => "/etc/logstash/certs/{{ ansible_hostname }}.p12"
```

**Comment by widhalmt** [comment]
@Nathaniel-Donahue how about that?

**Comment by martialblog** [comment]
I think this was made obsolete by https://github.com/NETWAYS/ansible-role-logstash/pull/99

**Pull request closed without merging**